setup.py

```diff
@@ -5,7 +5,7 @@
 
 setuptools.setup(
     name="securedrop-proxy",
-    version="0.1.2",
+    version="0.1.3",
```

Comment:
the other place we'll want to bump the version is in the proxy `Response` here: https://github.com/freedomofpress/securedrop-proxy/blob/master/securedrop_proxy/proxy.py#L21